Formula/sbt.rb

```diff
@@ -16,7 +16,7 @@ def install
       s.gsub! "/etc/sbt/sbtopts", "#{etc}/sbtopts"
     end
 
-    libexec.install "bin", "lib"
+    libexec.install "bin"
```

Comment:
sbt 1.3.13 lacks `lib` because there's an on-going issue with the preloaded local, but in the future `lib/` might come back.

Reply:
Oh maybe we should add a comment then